src/WorkspaceHelper.php

```diff
@@ -280,7 +280,9 @@ public static function isAllowed($element, Configuration $configuration, string
                 $parent = $parent->getParent();
             }
         }
-        $parentIds[] = $element->getId();
+        if (!is_null($element->getId())) {
```

Comment:
```suggestion
        if ($element->getId()) {
```
I think this should be enough, right? In older Pimcore Version it can be an 0 integer.